SpatialGDK/Extras/schema/actor_set_member.schema

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+package unreal;
+
+component ActorSetMember {
+	id = 9950;
```

Comment:
Any reason you started here, and not continued counting down from the last component id?